search_results_list.html

```diff
@@ -0,0 +1,34 @@
+<main class="main" role="main">
+  <div class="wrap clearfix">
+    <div class="row recipe-list">
+      <section class="content">
+        <div class="entries row">
+          <!-- Recipe grid square -->
+          {{#each posts}}
+            <div class="entry one-third" style="margin-bottom: 15px">
+              <!-- Recipe picture -->
+              <figure style="height: 169px; overflow: hidden;">
+                <img src="{{ ../baseurl }}/img/recipes/{{ this.title_image }}" alt="{{ this.title }}" />
```

Comment:
It'd be nice to do this the way it's done elsewhere - `{{ this.title_image | prepend: 'img/recipes'  | prepend: site.baseurl }}`, though we're now doing this in 3 places and should consider adding it to the whole-site config